rbd/pool_nautilus_test.go

```diff
@@ -30,6 +30,24 @@ func TestPoolMetadata(t *testing.T) {
 		assert.Error(t, err)
 	})
 
+	t.Run("NullKey", func(t *testing.T) {
```

Comment:
I do not know what the expectations are of a key being `""`. Setting an empty key does not make much sense, I think.

Reply:
This is just to prove that ceph functions handle gracefully for empty key and we don't need an explicit check in *PoolMetadata() functions.

Reply:
BTW to my surprise, SetMetadata takes empty key and doesn't return an error if you pass that :)